test/lightning/workflows/query_test.exs

```diff
@@ -60,21 +61,36 @@ defmodule Lightning.Workflows.QueryTest do
           enabled: false
         })
 
-      job = insert(:job, enabled: true, workflow: trigger.workflow)
+      job = insert(:job, workflow: trigger.workflow)
 
       insert(:edge, %{
         source_trigger: trigger,
         target_job: job,
-        workflow: job.workflow
+        workflow: job.workflow,
+        enabled: false
       })
 
       _disabled_cronjob =
-        insert(:job, enabled: false, workflow: trigger.workflow)
+        insert(:job, workflow: trigger.workflow)
+
+      insert(:edge, %{
+        source_trigger: trigger,
+        target_job: _disabled_cronjob,
+        workflow: _disabled_cronjob.workflow,
+        enabled: true
+      })
 
       webhook_trigger = insert(:trigger, type: :webhook)
 
       _non_cron_job =
-        insert(:job, enabled: true, workflow: webhook_trigger.workflow)
+        insert(:job, workflow: webhook_trigger.workflow)
+
+      insert(:edge, %{
+        source_trigger: webhook_trigger,
+        target_job: _non_cron_job,
+        workflow: _non_cron_job.workflow,
```

Comment:
Variables prefixed with an underscore are a hint that the variable isn't supposed to be used: https://hexdocs.pm/elixir/main/naming-conventions.html#underscore-_foo

It was left there as a 'comment' of sorts, in tests they are permissible but we try and not have these kinds of variable names in the application code.

So in this situation, could you remove the `_` prefix from the variable.